docs/application/dotnet/guides/nui/textlabel.md

```diff
@@ -180,6 +175,7 @@ For text decorations, the [Tizen.NUI.BaseComponents.TextLabel](https://samsung.g
 | `Ellipsis` |   `bool` |      Enable or disable ellipsis, if required. |
 | `AutoScrollLoopDelay` | `float` |   Auto scroll loop delay. |
 | `AutoScrollStopMode` | `AutoScrollStopMode` | Auto-scroll stop mode |
+| `MatchSystemLanguageDirection` | `bool` | The text alignment to match the direction of the system language |
```

Comment:
Kindly put a period at the end of the sentence. 

Suggested edit:
.... the direction of the system language.

Period is also missing in Line on. 177. Kindly fix if that comes under your scope.